src/commandline_options.cpp

```diff
@@ -205,9 +205,9 @@ commandline_options::commandline_options(const std::vector<std::string>& args)
 		("unsafe-scripts", "makes the \'package\' package available to Lua scripts, so that they can load arbitrary packages. Do not do this with untrusted scripts! This action gives ua the same permissions as the Wesnoth executable.")
 		("usercache-dir", po::value<std::string>(), "sets the path of the cache directory to $HOME/<arg> or My Documents\\My Games\\<arg> for Windows. You can specify also an absolute path outside the $HOME or My Documents\\My Games directory. Defaults to $HOME/.cache/wesnoth on X11 and to the userdata-dir on other systems.")
 		("usercache-path", "prints the path of the cache directory and exits.")
-		("userconfig-dir", po::value<std::string>(), "sets the path of the user config directory to $HOME/<arg> or My Documents\\My Games\\<arg> for Windows. You can specify also an absolute path outside the $HOME or My Documents\\My Games directory. Defaults to the userdata-dir.")
+		("userconfig-dir", po::value<std::string>(), "sets the path of the user config directory. You must specify an absolute path outside the $HOME or My Documents\\My Games directory. Defaults to the userdata-dir.")
 		("userconfig-path", "prints the path of the user config directory and exits.")
-		("userdata-dir", po::value<std::string>(), "sets the path of the userdata directory to $HOME/<arg> or My Documents\\My Games\\<arg> for Windows. You can specify also an absolute path outside the $HOME or My Documents\\My Games directory.")
+		("userdata-dir", po::value<std::string>(), "sets the path of the userdata directory. You must specify an absolute path outside the $HOME or My Documents\\My Games directory.")
```

Comment:
```suggestion
		("userdata-dir", po::value<std::string>(), "sets the path of the userdata directory. You must specify an absolute path.")
```

But this is not correct anyway is it? You can also specify a path starting with `.\` or `..\`.